service-self-backfill/src/main/kotlin/app/cash/backfila/service/selfbackfill/BackfilaSelfBackfillDevelopmentService.kt

```diff
@@ -0,0 +1,80 @@
+package app.cash.backfila.service.selfbackfill
+
+import app.cash.backfila.client.BackfilaDefaultEndpointConfigModule
+import app.cash.backfila.dashboard.ViewLogsUrlProvider
+import app.cash.backfila.service.BackfilaConfig
+import app.cash.backfila.service.BackfilaServiceModule
+import app.cash.backfila.service.persistence.DbBackfillRun
+import misk.MiskApplication
+import misk.MiskCaller
+import misk.MiskRealServiceModule
+import misk.environment.DeploymentModule
+import misk.hibernate.Session
+import misk.inject.KAbstractModule
+import misk.jdbc.DataSourceClusterConfig
+import misk.jdbc.DataSourceClustersConfig
+import misk.jdbc.DataSourceConfig
+import misk.jdbc.DataSourceType
+import misk.security.authz.DevelopmentOnly
+import misk.security.authz.FakeCallerAuthenticator
+import misk.security.authz.MiskCallerAuthenticator
+import misk.web.MiskWebModule
+import misk.web.WebConfig
+import misk.web.dashboard.AdminDashboardModule
+import wisp.deployment.Deployment
+
+internal fun main(args: Array<String>) {
```

Comment:
a new dev app just to test self backfilling, since we can't dep from :service to this module